_posts/2019-05-06-live-view-with-presence.md

```diff
@@ -0,0 +1,587 @@
+---
+author: Sophie DeBenedetto
+author_link: https://github.com/sophiedebenedetto
+categories: til
+date: 2019-05-06
+layout: post
+title:  Tracking Users in a Chat App with LiveView, PubSub Presence
+excerpt: >
+  Use Phoenix Presence in your LiveView to track user state with just a few lines of code.
+---
+
+After playing with LiveView and leveraging Phoenix PubSub to broadcast messages to all of a live view's clients, I wanted to try incorporating Phoenix Presence to track the state of these clients. So this past weekend I built a chat app using Phoenix LiveView, PubSub and Presence. The LiveView clocks in at 90 lines of code, and I was able to get the Presence-backed features up and running in no time! Keep reading to see how it works.
+
+## The App
+
+The chat app is fairly straightforward, and we won't get into the details of setting up LiveView in our Phoenix app here. You can check out the [source code](https://github.com/SophieDeBenedetto/phat/tree/tutorial) along with this [earlier post](https://elixirschool.com/blog/phoenix-live-view/) on getting LiveView up and running for more info.
+
+### Following Along
+
+If you'd like to follow along with this tutorial, clone down the repo [here](https://github.com/SophieDeBenedetto/phat/tree/tutorial) and follow the README instructions to get up and running. The starting state of the tutorial branch includes the chat domain model, routes, controller and the initial state of the LiveView, described below. You can also check out the completed code [here](https://github.com/SophieDeBenedetto/phat).
```

Comment:
Do you mind transferring this Repo into the org like we've done with other tutorial repos?